lxd/main_nsexec.go

```diff
@@ -99,6 +99,23 @@ int dosetns(int pid, char *nstype) {
 	return 0;
 }
 
+int dosetns_file(char *file, char *nstype) {
+	__do_close_prot_errno int mntns = -EBADF;
```

Comment:
That should probably be named `ns_fd` instead of `mntns`.

Reply:
@brauner will apply same change to dosetns function above I based this on.